docs/rook.md

```diff
@@ -1,16 +1,16 @@
 # Rook
 
-Rook is parent command which requires sub-command to get working. Currently, rook supports these sub-commands:
+Rook is a parent command and  requires sub-command to get working. Currently, rook supports the following  sub-commands:
 
-1. `purge-osd <osd-id> [--force]` : [purge osd](#purge-osd) permanently remove an OSD from the cluster. Multiple OSDs can be removed with a comma-separated list of IDs.
+1. `purge-osd <osd-id> [--force]` : [purge osd](#purge-osd) permanently remove an OSD from the cluster. Multiple OSDs can be removed in a single command with a comma-separated list of IDs.
 2. `version`: [version](#version) prints the rook version.
 3. `status`      : [status](#status) print the phase and conditions of the CephCluster CR
 4. `status all`  : [status all](#status-all) print the phase and conditions of all CRs
 5. `status <CR>` : [status  cr](#status-cr-name) print the phase and conditions of CRs of a specific type, such as 'cephobjectstore', 'cephfilesystem', etc
 
-## Purge OSD
+## Purge OSDs
 
-Permanently remove an OSD from the cluster. Multiple OSDs can be removed with a comma-separated list of IDs.
+Permanently remove an OSD from the cluster. Multiple OSDs can be removed in one invocation with a comma-separated list of IDs.
```

Comment:
How about moving the comment about multiple OSDs below the example? The example is only for a single OSD.
```suggestion
Permanently remove an OSD from the cluster. 

(below the output)
Multiple OSDs can be removed in one invocation with a comma-separated list of IDs.
```